packages/web/src/features/translation/TranslateWord.tsx

```diff
@@ -91,11 +110,15 @@ const TranslateWord = forwardRef<TranslateWordRef, TranslateWordProps>(
       );
     }, [glossWidth, hasMachineSuggestion]);
 
+    const hasTranslatorNote =
+      !!notesQuery.data?.data?.translatorNotes[word.id].content;
+    const hasFootnote = !!notesQuery.data?.data?.footnotes[word.id].content;
+
     return (
       <li ref={root} dir={originalLanguage === 'hebrew' ? 'rtl' : 'ltr'}>
         <div
           id={`word-${word.id}`}
-          className={`h-8 cursor-pointer font-mixed ${
+          className={`flex items-center h-8 cursor-pointer font-mixed ${
```

Comment:
```suggestion
          className={`flex items-center gap-1 h-8 cursor-pointer font-mixed ${
```

Maybe we should add `gap-2` or `gap-1` here, so that the button isn't immediately beside the text. Fiddle around with it to figure out which size of gap looks best.

![image](https://github.com/globalbibletools/gbt/assets/17209828/7e857955-3989-4b6c-b94c-555737670678)

Reply:
@Pertempto I've settled right in the middle with `gap-1.5` :)